geonode/qgis_server/tests/test_views.py

```diff
@@ -516,7 +516,33 @@ def test_add_delete_style(self):
         default_style_url = reverse(
             'qgis_server:default-qml',
             kwargs={
-                'layername': layer.name})
+                'layername': layer.name,
+                'style_name': 'new_style'})
```

Comment:
This url should return default style, so you should not specify the style name.

Reply:
Also ```response = self.client.get(default_style_url)``` and the rest at the bottom should be directly below this line (because that was this test all about, checking that the new default style changed).

Basically your new test should be before ```layer.delete()```